packages/bot-web-ui/src/components/transactions/transactions.jsx

```diff
@@ -5,13 +5,12 @@ import { useNewRowTransition } from '@deriv/shared';
 import { PropTypes } from 'prop-types';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
-import Transaction from './transaction.jsx';
-import Download from './download.jsx';
-import { transaction_elements } from '../constants/transactions';
-import { connect } from '../stores/connect';
-import { contract_stages } from '../constants/contract-stage';
-import '../assets/sass/download.scss';
-import '../assets/sass/transactions.scss';
+import { Download, Transaction } from 'Components';
+import { contract_stages } from 'Constants/contract-stage';
+import { transaction_elements } from 'Constants/transactions';
+import { connect } from 'Stores/connect';
+import 'Components/download/download.scss';
```

Comment:
In general, could we make it like components where the `index.js` of each component folder imports the `.scss` file?